crates/flux-driver/src/callbacks.rs

```diff
@@ -148,7 +151,32 @@ struct CrateChecker<'genv, 'tcx> {
 
 impl<'genv, 'tcx> CrateChecker<'genv, 'tcx> {
     fn new(genv: GlobalEnv<'genv, 'tcx>) -> Self {
-        CrateChecker { genv, cache: QueryCache::load() }
+        let checker = CrateChecker { genv, cache: QueryCache::load() };
+        checker.encode_flux_items_in_lean();
+        checker
+    }
+
+    fn encode_flux_items_in_lean(&self) {
+        let mut fcx: FixpointCtxt<'_, '_, i32> = FixpointCtxt::new(self.genv, None, None);
```

Comment:
Going through the code, I realize now that we don't need a `FixpointCtxt`, we only need an `ExprEncodingCtxt` and a `SortEncodingCtxt`. That way, we don't need to make the `KvarGen` optional.